### DISCUSSION TITLE
MAME on the App Store: Permission to use MAME name and logo

### DISCUSSION THREAD

**Discussion by yoshisuga** [mamedev/discussions#136] [General]
Hi! I'm the maintainer of the MAME4iOS repo, the port/osd of MAME for iOS. As some of you know Apple has allowed emulators in the App Store since April, but have been rejecting MAME because it was a "spam" app due to its binary similarity with other apps who presumably use some of the MAME source code.

After a couple of appeals, I got a call from their App Review team saying that they overturned their rejection for the "spam" app guideline, but needed some more information before they can approve it.

One of the things is getting the authorization from the MAME Dev team to use the MAME name and iconography:

```
Your app's metadata contains content that is similar to third-party content, which may create a misleading association with another developer's app or intellectual property. 


Specifically, the app's name and icons includes references to MAME.


Next Steps


It would be appropriate to revise the app and metadata to remove this third-party content before resubmitting for review. 

If you have the necessary rights to distribute an app with this third-party content, attach documentary evidence in the App Review Information section in App Store Connect and reply to this message. 
```

From MAME's Contact page, Gregory Ember is the trademark holder of MAME, and I would formally like to request his permission to use the MAME logo and icon imagery for use in the iOS app that we want to distribute in the App Store.

The app will be completely free without in-app purchases or subscriptions, and the [source code is here](https://github.com/yoshisuga/MAME4iOS/tree/master). A small team (myself and @toddla) keep MAME relatively up-to-date with recent versions and we will continue to do so. We have been maintaining the port of MAME for iOS for the past 8+ years with the intent of preservation in allowing the many games MAME supports to be experienced on modern iOS devices.

Thank you,

Yoshi Sugawara
@yoshisuga

**Comment by rb6502** [comment]
You aren't getting any answers because we don't have any.  I'm not even sure the team approves of MAME being in any app stores (that would theoretically paint a large bulls-eye on us even for a free app), and there are complications with the official rights at the moment.

**Reply by yoshisuga** [reply]
That makes sense, thank you for responding and providing some additional context.
